specs/beacon-chain.md

```diff
@@ -632,7 +632,8 @@ A block can have 0 or more `AttestationRecord` objects
 For each one of these attestations:
 
 * Verify that `slot <= parent.slot` and `slot >= max(parent.slot - CYCLE_LENGTH + 1, 0)`
```

Comment:
Not in this PR, but just noticed that `parent` hasn't been defined.

Reply:
addressed here https://github.com/ethereum/eth2.0-specs/pull/86/commits/9b9cfae117bb1b30975cc90759d3a6c5f3c9fa64